File: android/testData/snapshots/projects/buildSrcWithComposite/android-included/app/src/main/java/com/test/composite1/MainActivity.java
package com.test.composite1;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        com.test.composite1.Test.doSomething();
    }
}

File: android-lang-databinding/testData/projects/bitmaps/src/test/langdb/bitmaps/MainActivity.kt
package test.langdb.bitmaps

import android.os.Bundle
import androidx.fragment.app.FragmentActivity
import androidx.navigation.NavController
import androidx.navigation.NavHost
import androidx.navigation.Navigation

class MainActivity : FragmentActivity(), NavHost {
    private lateinit var navController: NavController

    override fun getNavController() = navController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        navController = Navigation.findNavController(this, R.id.nav_host_fragment)
    }
}
